File: C/section03/S03E05.c
#include <stdio.h>

int main() {
    int meters, centimeters;

    printf("Please type the meters value: \n");
    scanf("%d", &meters);

    centimeters = meters * 100;

    printf("\nThe value of %d meters to centimeters is %d centimeters\n", meters, centimeters);

    return 0;
}

File: Java/src/br.com.geekuniversity.com.section03/S03e05.java
package br.com.geekuniversity.com.section03;

import java.util.Scanner;

public class S03e05 {
    public static void main(String[] args) {
        int centimeters, meters;

        Scanner userInput =  new Scanner(System.in);
        
        System.out.println("Please enter the value in meters: ");
        meters = userInput.nextInt();
        
        centimeters = meters * 100;

        System.out.println("\n"+ meters +" meters in centimeters is: " + centimeters);
        
        userInput.close();

    }
}